base-utils/bzip2/spec

```diff
@@ -1,5 +1,7 @@
 VER=1.0.8
-REL=2
-SRCS="tbl::https://sourceware.org/pub/bzip2/bzip2-$VER.tar.gz"
-CHKSUMS="sha256::ab5a03176ee106d3f0fa90e381da478ddae405918153cca248e682cd0c4a2269"
+REL=3
+SRCS="tbl::https://sourceware.org/pub/bzip2/bzip2-$VER.tar.gz \
+      file::rename=bzip2.pc::https://src.fedoraproject.org/rpms/bzip2/raw/rawhide/f/bzip2.pc"
```

Comment:
If you really want to download this file instead of bundling it, please use a "pinned version" URL instead of this.

For example:
```suggestion
      file::rename=bzip2.pc::https://src.fedoraproject.org/rpms/bzip2/raw/5141dcf62cb95a32f4541b3c417c60427d4d34af/f/bzip2.pc"
```

(Please ignore this comment if you decided to go with the bundling route)